backend/src/api/database-migration.ts

```diff
@@ -801,6 +801,323 @@ class DatabaseMigration {
       `);
       await this.updateToSchemaVersion(93);
     }
+
+    // Unify database schema for all mempool netwoks
+    // versions above 94 should not use network-specific flags
+    if (databaseSchemaVersion < 94) {
+
+      if (!isBitcoin) {
+        // Apply all the bitcoin specific migrations to non-bitcoin networks: liquid, liquidtestnet and testnet4 (!)
```

Comment:
for obscure reasons, `config.MEMPOOL.NETWORK` ought be set to "testnet" in the backend for both legacy testnet and testnet4, so both should have been treated correctly as bitcoin networks.